pkg/values/value.go

```diff
@@ -251,7 +251,13 @@ func unwrapTo[T any](underlying T, to any) error {
 		}
 		*tb = underlying
 	default:
-		return fmt.Errorf("cannot unwrap to value of type: %T", to)
+		rTo := reflect.ValueOf(to)
```

Comment:
why is this changing? a comment about intention would be useful

Reply:
type FeedId string

then decode to FeedId shouldn't break.

Reply:
This should have a corresponding test in the values package.